src/app/home/home.component.ts

```diff
@@ -412,17 +474,26 @@ export class HomeComponent implements AfterViewInit {
         this.hasStoppedTypingForEventualMarking = false;
     }
 
-
+    /**
+     * Marks the editor based on the lastly (time-wise) typed character. For example breaking the current line is
+     * considered as a signal to attempt to mark the current written text. Attempting to mark the editor after every
+     * keystroke can annoy the user and will also mean a significantly larger amount of requests made.
+     * @param $event
```

Comment:
add a comment on the event parameter- of which element does the element belong to and so on. Otherwise gives the impression of direct event handler on the html side when is in fact only used internally here.

Reply:
Added some more information to it